### PR TITLE
move checkDeps & showUpdates to piamenv

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
pikPiamPackages was renamed to piamPackages. checkDeps and showUpdates are removed in this PR, because they moved to piamenv (https://github.com/pik-piam/piamenv/pull/1). This API change will break things (remind for sure), that's why this is a draft PR. I will only turn this into a normal PR once nothing will break when this is merged.

### CODE REVIEW

**Comment by pfuehrlich-pik** [comment]
Once https://github.com/remindmodel/remind/pull/1093 is merged this can be merged too, pikPiamPackages, showUpdates, and checkDeps are not used anywhere besides remindmodel, lucode2, and piamenv according to reposearch.

**Comment by pfuehrlich-pik** [comment]
ready for review @mikapfl

File: NAMESPACE [hunk 1]
```diff
@@ -28,7 +27,7 @@ export(mergestatistics)
 export(package2readme)
 export(packageInfo)
 export(path)
-export(pikPiamPackages)
+export(piamPackages)
```
Comment (mikapfl) [inline NAMESPACE:30]: why the name change?

**Review by mikapfl — APPROVED** [review]

File: NAMESPACE (hunk 1 above)
Comment (pfuehrlich-pik) [reply NAMESPACE:30]: pik piam is kinda a little bit redundant (both Ps stand for Potsdam), also it's more consistent with piamenv, also it's shorter.

**Pull request merged**